RELEASE-NOTES.md

```diff
@@ -7,7 +7,7 @@
 + Fix bug in the computation of the log pseudolikelihood values (SMC-ABC). (see [#4672](https://github.com/pymc-devs/pymc3/pull/4672)).
 
 ### New Features
-+ BART with non-gaussian likelihoods (see [#4675](https://github.com/pymc-devs/pymc3/pull/4675)).
++ BART with non-gaussian likelihoods (see [#4675](https://github.com/pymc-devs/pymc3/pull/4675) and [#4709](https://github.com/pymc-devs/pymc3/pull/4709)).
```

Comment:
Did you remove the gaussian likelihood or that old functionality is still present?

Reply:
That is still present, you can now have other likelihoods too. Actually "non-Gaussian" is probably not a very good description, as likelihoods like Student-T were already possible. The problem was more about the likelihood being bounded vs unbounded.